patients/repo.go

```diff
@@ -578,11 +914,15 @@ func isSortAttributeValid(attribute string) bool {
 	return ok
 }
 
-func generateListSortStage(sort *store.Sort) bson.D {
+func generateListSortStage(sorts []*store.Sort) bson.D {
```

Comment:
the clients can still only sort on one field, but it was a requirement to sort empty last, instead of first, and this uses a 2nd sort key conditionally inserted on the eligible sort keys.